receiver/mongodbreceiver/scraper.go

```diff
@@ -22,6 +24,8 @@ import (
 	"github.com/open-telemetry/opentelemetry-collector-contrib/receiver/mongodbreceiver/internal/metadata"
 )
 
+const defaultMongodbPort = 27017
```

Comment:
There's an existing `mongoPort` variable in `integration_test.go` that we can remove as a result of this addition. (Since the `integration_test.go` variable is hidden behind the `integration` build flag, it would be better to use this one.)

Also, could we use this variable [when creating the default config](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/93a78c6ad8f57c5e4ff29a5f41779af03fc85145/receiver/mongodbreceiver/factory.go#L34)?

Reply:
That's a string

Reply:
Right, but we could just create a temporary string variable and use it instead. Not a big deal, just trying to deduplicate wherever possible.

```
defaultEndpoint := fmt.Sprintf("%s:%v", "localhost", defaultMongodbPort)
```

Reply:
In both cases, we need port, not endpoint: int64 and string. I wouldn't personally bother with the conversion here. I don't think it simplifies things or reduces the code.

Reply:
Oh, I was talking about the the integration tests only where we define `mongoPort`. The `defaultEndpoint` probably makes sense. Let me apply that

Reply:
That's fair, I didn't realize `mongoPort` was a string. This is fine with me.

Reply:
> Oh, I was talking about the integration tests only where we define mongoPort. The defaultEndpoint probably makes sense. Let me apply that

Yeah, sorry, I misinterpreted your initial response to be about the second part of my comment, and my `defaultEndpoint` comment was responding to that part. 🙂